specification/patient-care-aggregator-api-appointment-standard.yaml

```diff
@@ -76,32 +76,509 @@ x-spec-publication:
   try-this-api:
     disabled: true
 servers:
-  - url: 'https://tbc'
-    description: Sandbox environment.
-  - url: 'https://tbc'
-    description: Integration test environment.
-  - url: 'https://tbc'
+  - url: 'https://integration.servita-demo.co.uk'
+    description: Connectivity and data quality test environment.
+  - url: 'https://aos.patient-care-aggregator.com'
+    description: End-to-end test environment.
+  - url: 'https://prod.patient-care-aggregator.com'
     description: Production environment.
 paths:
-  /tbc:
+  /fhir/R4/Appointment:
     get:
-      summary: Get appointments
-      operationId: tbc
-      description: |
-        TBC
+      security:
+        - bearerAuth: []
+      tags:
+      - GET Appointments
+      summary: "Note: this is an example path. You may use a different path if required."
       parameters:
-        - name: tbc
-          in: query
-          description: |
-            TBC
-          required: true
-          schema:
-            type: string
-            example: "TBC"
+      - name: Authorization
+        in: header
+        required: true
+        description: 'Authorization Header. The NHS Login ID Token passed as a Bearer token. This is generated when the user logs into NHS App via NHS Login as passed to this API for validation and signature verification.'
+        schema:
+          type: string
+      - name: x-correlation-id
+        in: header
+        required: true
+        description: 'Header parameter. A GUID, e.g. 8772a09a-8d55-4514-8d8b-a1526055c599, that is logged by all the Wayfinder services that process the request. Used for end-to-end debugging.'
+        schema:
+          type: string
+      - name: patient:identifier
+        in: query
+        required: true
+        description: 'Query parameter. The NHS Number of the patient, passed as a FHIR patient identifier, e.g. patient:identifier=https://fhir.nhs.uk/Id/nhs-number|9726779057. '
+        schema:
+          pattern: ^https:\/\/fhir\.nhs\.uk\/Id\/nhs-number\|[1-9][0-9]{9}$
+          type: string
       responses:
-        '2XX':
-          description: |
-            TBC
-        '4XX':
-          description: |
-            TBC
+        200:
+          description: Successful
+          content:
+            'application/json':
+              schema:
+                description: A FHIR bundle containing results of the request.
```

Comment:
Maybe add a comment to explain to return a 200 even if there are no appointments for the patient?

Reply:
Already mentioned as to do on https://nhsd-confluence.digital.nhs.uk/display/WAYF/Secondary+care+booking+system+requirements